src/tests/functional/plugin/shared/include/base/behavior_test_utils.hpp

```diff
@@ -10,6 +10,7 @@
 #include "common_test_utils/file_utils.hpp"
 #include "openvino/util/file_util.hpp"
 #include "functional_test_utils/summary/api_summary.hpp"
```

Comment:
@sbalandi Could you please review the file and get rid OV1.0 based test classes?